src/sentry/search/snuba/backend.py

```diff
@@ -64,6 +71,14 @@ def first_release_all_environments_filter(version, projects):
     )
 
 
+def inbox_filter(inbox):
+    # TODO this should probably take projects for perf reasons...should I add extra stuff to filter on to the GroupInbox model?
+    query = Q(id__in=GroupInbox.objects.all().values_list("group_id", flat=True))
```

Comment:
This seems like something we won't want to release with.

Reply:
Yeah, probably should have project id here so this will be faster.

I'm not sure if the `__in` is the best query here. I wonder how many items we're likely to have in inbox per project...  I'm also wondering if a join might work better, something like`groupinbox__project=project`, maybe? We'd have to profile to check. Probably ok to use the `__in` for now.

Reply:
Could also be worth just throwing `organization_id` on here as well

Reply:
Should this be a subquery condition or a join? This could load a pile of records up only to send them back to the database as parameters.